perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -112,10 +112,17 @@ sub send() {
                     ( $last_problem_state eq '' && $row->{item_problem_state} eq 'confirmed' ) ||
                     $last_problem_state ne $row->{item_problem_state}
                 ) ) {
-                    my $update = sprintf(
-                        _('Problem marked as %s'),
-                        FixMyStreet::DB->resultset("State")->display($row->{item_problem_state}, 1)
-                    );
+                    my $state = '';
+                    if ($row->{update_state} eq 'not responsible') {
```

Comment:
Should we move this into resultset("State")->display so it doesn't have to be duplicated?

Reply:
👍